lib/api/scripts/generateVersion.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
```

Comment:
I wasn't able to load `package.json` in typescript, since it's outside the `./src` directory.

Generating a `version.ts` upon build felt like an OK solution.

If there's a way to do this in typescript more easily, let me know.

Reply:
I've tried to remove rootDir and import package.json. Tests passed, but it broke build somehow. Let it be with version generator..